packages/edit-site/src/components/left-sidebar/navigation-panel/style.scss

```diff
@@ -3,6 +3,7 @@
 	height: 100%;
 	position: relative;
 	width: $nav-sidebar-width;
+	background: #1e1e1e;
```

Comment:
It would be great to put this somewhere as a variable. Like we did with `nav-sidebar-width`. Which file would be the place for it?

Reply:
Don't we already have a base variable for this in `darkGray.primary` and `$gray-900`?

Reply:
We do, but if we change the Navigation component's background color, then we need to change it everywhere else. That's why a variable would be handy. This background should match the Navigation's component background all the time.